src/components/utils/inputDocumentTypes.js

```diff
@@ -0,0 +1,9 @@
+export const documentTypes = [
```

Comment:
Use this as input type validation (validator: type =>documentTypes.includes(type))
Remember to change the name of this const to refer to the new component name